plugins/azure/keyvaults/keyExpirationEnabled.js

```diff
@@ -51,7 +51,7 @@ module.exports = {
                         
                         if (key.attributes) {
                             let attributes = key.attributes;
-                            if (attributes.expires && attributes.expires !== null && attributes.expires !== '') {
+                            if (attributes.exp && attributes.exp !== null && attributes.exp !== '') {
```

Comment:
```suggestion
                            if ((attributes.expires && attributes.expires !== null && attributes.expires !== '') || (attributes.exp && attributes.exp !== null && attributes.exp !== '')) {
```

Reply:
Just to keep it backwards compatible in case all the Azure APIs haven't upgraded